pkg/standalone/standalone.go

```diff
@@ -615,13 +650,14 @@ func runSchedulerService(wg *sync.WaitGroup, errorChan chan<- error, info initIn
 	} else {
 		osPort := 50006
 		if runtime.GOOS == daprWindowsOS {
-			osPort = 6060 //TODO confirm this
+			osPort = 6060
 		}
 
 		args = append(args,
 			"-p", fmt.Sprintf("%v:50006", osPort),
-			"-p", fmt.Sprintf("%v:8081", schedulerHealthPort), //TODO confirm this
-			"-p", fmt.Sprintf("%v:9091", schedulerMetricPort), //TODO confirm this
+			"-p", fmt.Sprintf("%v:2379", schedulerEtcdPort),
+			"-p", fmt.Sprintf("%v:8080", schedulerHealthPort),
+			"-p", fmt.Sprintf("%v:9090", schedulerMetricPort),
```

Comment:
I think I put 8081 and 9091 originally bc I ran into an error on competing ports, but if you didn't with your testing then this is fine

Reply:
The competing ports are the first ones ("os port"). The second ones are the ports in the container.